lib/internal/Magento/Framework/View/Helper/SecureHtmlRender/HtmlRenderer.php

```diff
@@ -49,10 +74,10 @@ public function renderTag(TagData $tagData): string
         }
 
         $html = '<' .$tagData->getTag() .$attributesHtml;
-        if ($content) {
-            $html .= '>' .$content .'</' .$tagData->getTag() .'>';
-        } else {
+        if (in_array($tagData->getTag(), self::VOID_ELEMENTS)) {
```

Comment:
Hi,

Could you replace the in_array statement with the isset due to performance issues?
https://gist.github.com/alcaeus/536156663fac96744eba77b3e133e50a

Reply:
Better re-check this case carefully!

Reply:
@ihor-sviziev Good point. I agree that searching by value is slower but in this case insignificant gain. Also php 7.* give slower result that's really interested

PHP 8.0
Function call in_array took 0.00493 seconds
Function call isset took 0.00360 seconds
Function call array_key_exists took 0.00388 seconds

PHP 7.4
Function call in_array took **0.00485** seconds
Function call isset took **0.00541** seconds
Function call array_key_exists took 0.00674 seconds

PHP 7.3
Function call in_array took **0.00530** seconds
Function call isset took **0.00563** seconds
Function call array_key_exists took 0.00464 seconds

Reply:
https://gist.github.com/Den4ik/ddfae8384bf8e26bd42af8c6e1662b9e

Reply:
@Den4ik tested your code locally and on the https://sandbox.onlinephpfunctions.com/
PHP 7.3 - isset is faster ✔
![image](https://user-images.githubusercontent.com/1873745/122919914-76050780-d369-11eb-8b00-6c0eabdeac54.png)
PHP 7.4 - isset is faster ✔
![image](https://user-images.githubusercontent.com/1873745/122919978-8917d780-d369-11eb-9186-20078087cab4.png)
PHP 8.0 - isset is faster ✔
![image](https://user-images.githubusercontent.com/1873745/122920118-b3699500-d369-11eb-8869-721da0c4d763.png)

Reply:
```suggestion
        if (isset(self::VOID_ELEMENTS_MAP[$tagData->getTag()])) {
```